frontend/src/v5/ui/components/viewer/cards/tickets/sequencingProperty/sequencingProperty.component.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { DateTimePickerProps, DateTimePicker } from '@controls/inputs/datePicker/dateTimePicker.component';
+import SequencingIcon from '@assets/icons/outlined/sequence-outlined.svg';
+import CloseIcon from '@assets/icons/outlined/close-outlined.svg';
+import { useContext, useEffect } from 'react';
+import dayjs from 'dayjs';
+import { SequencesActionsDispatchers } from '@/v5/services/actionsDispatchers';
+import { useFormContext } from 'react-hook-form';
+import { SEQUENCING_START_TIME, SEQUENCING_END_TIME } from '@/v5/ui/routes/viewer/tickets/tickets.constants';
+import { TicketContext } from '@/v5/ui/routes/viewer/tickets/ticket.context';
+import { formatMessage } from '@/v5/services/intl';
+import { Container, IconContainer, SequenceIconContainer } from './sequencingProperty.styles';
+
+const LABLES = {
```

Comment:
There's a typo with LABELS

Reply:
Can't believe this passed Dan's check. @The-Daniel, I'm very disappointed with you 😞😞